### ISSUE TITLE
Sound problem

### ISSUE THREAD

**Issue by feknall** [deepdrive/deepdrive-sim#28]
The simulator doesn't have any sound. When I go to settings, its value is 0 and I can't change it using a mouse or keyboard.
Ubuntu 18.04

**Comment by crizCraig** [comment]
Yeah, we removed the sound as it was annoying when constantly opening closing the sim during experiments. @dnsmernst would know how easy it would be to reenable. We use to have engine, horn, tire screeching noises, change gear sounds, etc... 

At some point sound will become important in self-driving, esp use of the horn IMHO, so this will become important in simulation as well. Although no one to my knowledge uses human audible sound as an input besides perhaps Waymo for their emergency vehicle detection. Tesla uses sonar but that's for active sensing of the surroundings.

**Comment by feknall** [comment]
Yes, you are right. I want to hear the sound because of its proper sense. It would be more realistic if we could listen to the sound, not because of the learning process, because of ourselves. I think enabling or disabling it using a config would be nice.

**Comment by crizCraig** [comment]
Agreed. To enable this, we should cross reference v2 which had sounds. The sound settings are in the Car blueprints. We'd just need to copy/re-enable them in the current version.
